spec.bs

```diff
@@ -165,3 +169,63 @@ The {{HTMLFencedFrameElement/src}} IDL attribute must [=reflect=] the respective
 <h3 id=dimension-attributes>Dimension attributes</h3>
 
 This section details monkeypatches to [[!HTML]]'s <a href="https://html.spec.whatwg.org/multipage/embedded-content-other.html#dimension-attributes">Dimension attributes</a> section. This section will be updated to include <{fencedframe}> in the list of elements that the <code>[=width=]</code> and <code>[=height=]</code> dimension attributes apply to.
+
+<h3 id=new-request-destination>New fenced-frame-src [[!CSP]] [=directive=]</h3>
+
+Fenced frames are a different element from an iframe. Therefore, using the <b><i>[=frame-src=]</i></b> directive wouldn't give web sites enough control over their CSP rules. Introduce a new [[!CSP]] [=directive=]: <b><i>fenced-frame-src</i></b>. The monkey-patched specification is printed below:
+
+<h4 id="directive-fenced-frame-src">`fenced-frame-src`</h4>
+
+The <dfn export>fenced-frame-src</dfn> directive restricts the URLs which may be loaded into
+<a>nested browsing contexts</a>. The syntax for the directive's name and value
+is described by the following ABNF:
+
+<pre>
+  directive-name  = "fenced-frame-src"
+  directive-value = <a grammar>serialized-source-list</a>
+</pre>
+
+<div id="fenced-frame-src example" class="example">
+  Given a page with the following Content Security Policy:
+  <pre>
+    <a http-header>Content-Security-Policy</a>: <a>fenced-frame-src</a> https://example.com/
+  </pre>
+
+  Fetches for the following code will return a network errors, as the URL
```

Comment:
Maybe let's explicitly link to https://fetch.spec.whatwg.org/#concept-network-error?